src/admin-views/widgets/list.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Admin View: List Widget
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/list.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @var array<array> $admin_fields An array of admin fields to display in the widget form.
+ *
+ * @version TBD
+ */
+
+use Tribe__Utils__Array as Arr;
+
+if ( empty( $admin_fields ) ) {
+	return;
+}
+
+foreach ( $admin_fields as $id => $field ) {
+
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $id

<sub>hash: 9a7b398a8813e54490e93cb442bb3c83</sub>